### DISCUSSION TITLE
How to keep date picker on screen

### DISCUSSION THREAD

**Discussion by ghost** [israel-dryer/ttkbootstrap#523] [General]
I have a ttkbootstrap DateEntry widget at the bottom of my screen. 
When I open the date picker it overlaps the edge of the screen. Could anyone suggest a solution to this, thanks.

**Comment by Megsman** [comment]
did you set the parent parameter?

**Comment by ghost** [comment]
Yes, I did.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
